### PR TITLE
feat(neon_talk): Add placeholder implementation

### PR DESCRIPTION
**Pull request by provokateurin**
Towards https://github.com/nextcloud/neon/issues/356
Extracted from https://github.com/nextcloud/neon/pull/366

### CODE REVIEW

File: packages/neon/neon_talk/lib/neon_talk.dart [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/// The Neon client for the Dashboard app.
+///
+/// Add `DashboardApp()` to your runNeon command to execute this app.
```
Comment (Leptopoda) [inline packages/neon/neon_talk/lib/neon_talk.dart:3]:
```suggestion
/// The Neon client for the Talk app.
///
/// Add `TalkApp()` to your runNeon command to execute this app.
```

File: packages/neon/neon_talk/lib/src/blocs/talk.dart [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import 'package:meta/meta.dart';
+import 'package:neon_framework/blocs.dart';
+import 'package:neon_framework/models.dart';
+
+/// Bloc for fetching Talk rooms
+sealed class TalkBloc implements InteractiveBloc {
+  /// Creates a new Talk Bloc instance.
+  @internal
```
Comment (Leptopoda) [inline packages/neon/neon_talk/lib/src/blocs/talk.dart:8]: why is the factory internal and not the entire class?

Comment (provokateurin) [reply packages/neon/neon_talk/lib/src/blocs/talk.dart:8]: This forces us to document the class as adding the internal annotation would no longer require docs.

Comment (provokateurin) [reply packages/neon/neon_talk/lib/src/blocs/talk.dart:8]: That's how it is done in all the other blocs as well.

**Review by Leptopoda — APPROVED** [review]

**Pull request merged**